kubernetes/zenko/charts/backbeat/values.yaml

```diff
@@ -10,7 +10,7 @@ global:
 
 image:
   repository: zenko/backbeat
-  tag: 8.2.2
+  tag: 8.2.3
```

Comment:
@tcarmet I think you need 8.2.4 image right?

Reply:
right! Thought I did it, but I messed up my git commands, it's fixed now, thanks!